exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,101 @@
+{
+   "book-store": {
+      "description": [
+                       "Calculate lowest price for shopping basket only ",
+                       "containing books from a single series"
+                     ],
+      "cases": [
+         {
+            "description": "A basket containing only a single book.",
+            "basket": [1],
+            "targetgrouping": [1],
+            "expected": 8.0
+         },
+         {
+            "description":  "A basket containing only two of the same book.",
+            "basket": [2, 2],
+            "targetgrouping": [[2], [2]],
+            "expected": 16.0
+         },
+         {
+            "description": "No charge to carry around an empty basket.",
+            "basket": [],
+            "targetgrouping": [],
+            "expected": 0.0
+         },
+         {
+            "description": "A basket containing only two different books.",
+            "basket": [1, 2],
+            "targetgrouping": [1, 2],
+            "expected": 15.2
+         },
+         {
+            "description": "A basket of three different books.",
+            "basket": [1, 2, 3],
+            "targetgrouping": [1, 2, 3],
+            "expected": 21.60
+         },
+         {
+            "description": "A basket of four different books.",
+            "basket": [1, 2, 3, 4],
+            "targetgrouping": [1, 2, 3, 4],
+            "expected": 25.60
+         },
+         {
+            "description": "A basket of five different books.",
+            "basket": [1, 2, 3, 4, 5],
+            "targetgrouping": [1, 2, 3, 4, 5],
+            "expected": 30.00
+         },
+         {
+            "description": [
+                            "A basket containing eight books consisting of a ",
```

Comment:
I am making two changes to address that having more than any one book in a group is of no advantage to the customer and the other change to draw the readers attention to this special case, that is counter-intuitive to how one might group the books, as you have pointed out.